egs2/vivos/asr1/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+stage=1
+stop_stage=11
+nj=16
+decode_asr_model=eval.loss.best.pth
+
+train_set="train_nodev"
+train_dev="train_dev"
+eval_set="test"
+
+asr_config=conf/train.yaml
+decode_config=conf/decode.yaml
+
+lm_config=conf/lm_char.yaml
+use_lm=true
+use_wordlm=false
+word_vocab_size=7184
+
+nlsyms_txt=data/nlsyms.txt
+
+./asr.sh                                        \
+    --stage ${stage}                            \
+    --stop_stage ${stop_stage}                  \
+    --nj ${nj}                                  \
+    --decode_asr_model ${decode_asr_model}      \
+    --feats_type fbank_pitch                    \
```

Comment:
Could you test with feats_type=raw? It indicates on-the-fly feature extraction using librosa.

Basically we shouldn't depend on Kaldi command as possible  and `raw` is standard method in epsnet2.
 `fbank_pitch` is manly put for reproducing the previous result and also as an example to use the extracted features from external system.

Reply:
I also trained and tested with `feats_type=raw` however I used `fbank_pitch` as default to reproduce espnet1 results. Do you want me to change defaults and results?

@sw005320 @kamo-naoyuki Should I try to reproduce recipes closely or I should use new defaults and revisit (a bit) recipes?

Reply:
Thank you. Yes, it's better to change the default to feats_type=raw if there is not significant degradation. 

I think we do not always need to reproduce the result even if small degradation is observed.
Now is good time to revisit recipes if you have better idea.

Reply:
OK! I'll update recipe/results after retraining, I erased exp folders for raw type yesterday..

Reply:
| dataset | feats_type | CER |
| --- | --- | --- |
| dev (no LM) | fbank_pitch | 19.0 |
| dev (LM) | fbank_pitch | 17.7 |
| dev (no LM) | raw | 20.9 |
| dev (LM) | raw | 19.0 |
| test (no LM) | fbank_pitch | 19.7 |
| test (LM) | fbank_pitch | 18.0 |
| test (no LM) | raw | 20.8 |
| test (LM) | raw | 18.5 |

Is the degradation ok for you?

Reply:
hmm, what do you think @sw005320 ?

Reply:
I think it's OK, but I found that I saw a significant degradation in the WER without LM. I don't know what happened.

Reply:
Yes I observed the same thing. However CER is on par with reported results in `egs/` and WER isn't reported so I didn't think there was a problem.

Reply:
But https://github.com/espnet/espnet/blob/49d471bfa359aff182c58a65a73cb63891215001/egs2/vivos/asr1/README.md#asr_train_fbank_pitch_char looks normal???

Reply:
Thanks, I found the problem! This WER was related to the first bullet in my original post, I did a mistake when parsing folders. Updated!